Coverage-x86/Coverage-x86.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Right, so this will only work in VS2017. 

I would have preferred a separate vcxproj / sln file for VS2017 and a separate vcxproj / sln for VS2015. Regardless, we plan to move to VS2017 somewhere in the next months anyways so I guess I can work with this.